merkle_patricia_tree/merkle_tree/src/main.rs

```diff
@@ -209,9 +204,9 @@ fn add_sub(input: &mut [u8]) -> Vec<u8> {
     // signed integer
     let intsize = match remain {
         1..=1                => 1,
-        2..=3                => 1 << rand_number_in_range(0,1),
-        4..=7                => 1 << rand_number_in_range(0,2),
-        8..=core::usize::MAX => 1 << rand_number_in_range(0,3),
+        2..=3                => 1 << rand_number_in_range(0,1, seed),
```

Comment:
This can only return `0`, as `0..1` (i.e. `[0, 1)`) only contains the number `0`.

Reply:
fixed!